backend/Repositories/SightingRepo.cs

```diff
@@ -51,5 +54,27 @@ public IEnumerable<Sighting> GetSightingsBySpeciesId(int speciesId)
                 .Where(s => s.ConfirmationStatus == ConfirmationStatus.Approved)
                 .OrderByDescending(s => s.SeenOn);
         }
+        public bool ConfirmRequest(int SightingId)
+        {
+            var sighting = _context.Sightings.Find(SightingId);
+            if (sighting != null)
+            {
+                sighting.ConfirmationStatus = ConfirmationStatus.Approved;
+                _context.SaveChanges();
+                return true;
+            }
+            return false;
+        }
+        public bool RejectRequest(int SightingId)
+        {
+            var sighting = _context.Sightings.Find(SightingId);
+            if (sighting != null)
+            {
+                sighting.ConfirmationStatus = ConfirmationStatus.Rejected;
+                _context.SaveChanges();
+                return true;
+            }
+            return false;
+        }
```

Comment:
These are fantastic! But, instead of returning a `bool`, I'd like them to return _the new, updated Sighting_:
```suggestion
        public Sighting ConfirmRequest(int SightingId)
        {
            var sighting = _context.Sightings.Find(SightingId);
            if (sighting != null)
            {
                sighting.ConfirmationStatus = ConfirmationStatus.Approved;
                _context.SaveChanges();
                return sighting;
            }
            return false;
        }
        public Sighting RejectRequest(int SightingId)
        {
            var sighting = _context.Sightings.Find(SightingId);
            if (sighting != null)
            {
                sighting.ConfirmationStatus = ConfirmationStatus.Rejected;
                _context.SaveChanges();
                return sighting;
            }
            return false;
        }
```

and if you can't find the Sighting object, throw an exception! In this case an `ArgumentException` is probably the right choice:

```suggestion
        public bool ConfirmRequest(int SightingId)
        {
            var sighting = _context.Sightings.Find(SightingId);
            if (sighting != null)
            {
                sighting.ConfirmationStatus = ConfirmationStatus.Approved;
                _context.SaveChanges();
                return true;
            }
            throw new ArgumentException($"The sighting with ID {SightingId} could not be found");
        }
        public bool RejectRequest(int SightingId)
        {
            var sighting = _context.Sightings.Find(SightingId);
            if (sighting != null)
            {
                sighting.ConfirmationStatus = ConfirmationStatus.Rejected;
                _context.SaveChanges();
                return true;
            }
            throw new ArgumentException($"The sighting with ID {SightingId} could not be found");
        }
```